### ISSUE TITLE
Can't use LiveStyle with a locally stored file

### ISSUE THREAD

**Issue by VonBasilic** [livestyle/issues#130]
Hi,

I just discovered LiveStyle and find it amazing.
However it seems I can't use it with my project. I'm working with HTML and CSS files all stored locally on my hard drive, and when I open my HTML file in Chrome (see the local address in the address bar), it's the only tab that I can't use the Chrome LiveStyle extension with. See the screenshot.

![capture d ecran 2016-07-05 a 18 27 32](https://cloud.githubusercontent.com/assets/20303066/16589952/2f0a9744-42de-11e6-865d-e4806ecadc06.png)

**Comment by VonBasilic** [comment]
Address of my file in the Chrome address bar is :
file:///users/Baptiste/Documents/Skeleton-2.0.4/PPP/PPP%20skeleton.html
so it's a local file

and here is the screenshot for a "normal" web page, where LiveStyle works perfectly well

![capture d ecran 2016-07-05 a 18 25 59](https://cloud.githubusercontent.com/assets/20303066/16589924/0eb9b7c2-42de-11e6-8aa1-e65223800db2.png)

**Comment by sergeche** [comment]
Yes, there’s a random issue with latest Chrome: for some reason in looses connection with `file://` page which, in turn, looks like a common scenario when extension is updated. It will be fixed in next release (but no ETA yet)

**Comment by VonBasilic** [comment]
Ok thank you for your quick answer.
What is the workaround? Should I set up a local host? Would that make it work?

**Comment by sergeche** [comment]
Yes, creating a local web-server (e.g. using `http://` scheme) should work

**Comment by VonBasilic** [comment]
Thank you

**Comment by BaptisteVan** [comment]
Thank you

On Tue, Jul 5, 2016 at 6:43 PM Sergey Chikuyonok <notifications@github.com>
wrote:

> Yes, creating a local web-server (e.g. using http:// scheme) should work
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/livestyle/issues/issues/130#issuecomment-230517030>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ATXM2guvlnEJneFBEb3zmvV-Xr3hcXLnks5qSnuTgaJpZM4JFPC_>
> .
>
-- 
*Baptiste Vanpoulle*
baptiste.vanpoulle@gmail.com
+33 6 27 96 83 44
